src/components/TransactionsList.tsx

```diff
@@ -1,59 +1,78 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./TransactionsList.css";
-import viewIcon from "/assets/view.svg";
 import { useTranslation } from "react-i18next";
+import { BPDTransactionList } from "../../generated/definitions/BPDTransactionList";
+import { BPDTransaction } from "../../generated/definitions/BPDTransaction";
+import { GetBPDTransactionsT } from "../../generated/definitions/requestTypes";
+import { fromEither, fromLeft, tryCatch } from "fp-ts/lib/TaskEither";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { TypeofApiResponse } from "italia-ts-commons/lib/requests";
+import { BackofficeClient } from "../helpers/client";
+import { Transaction } from "./Transaction";
+import { toError } from "fp-ts/lib/Either";
+import { getCitizenId, getUserToken } from "../helpers/coredata";
+import { Location } from "history";
 
-const TransactionsList = () => {
-  function openDetail(index: number): void {
-    const idDetail = `detail${index}`;
-    document.getElementById(idDetail)?.classList.toggle("d-none");
-  }
+type Props = {
+  location: Location<{ citizenid: string }>;
+};
+
+export const TransactionsList: React.FunctionComponent<Props> = props => {
+  const [resultData, setResultdata] = useState<BPDTransactionList | undefined>(
+    undefined
+  );
   const { t } = useTranslation();
 
+  useEffect(() => {
+    // TaskEither
+    // tslint:disable-next-line: no-floating-promises
```

Comment:
this is a code smell, you sould add a .catch() clause after .run() instead of suppressing the hint
otherwise you'll get uncaught promise exceptions in case of errors